docs/server-side/v2/changelog/release-notes.mdx

```diff
@@ -4,6 +4,10 @@ nav: 5.1
 ---
 
 This Changelog highlights notable changes to the 100ms server-side API, such as API additions, improvements, and deprecations. Also, we've included developer experience improvements to this page to keep you on track with items that will enhance your integration journey.
+## 2023-04-28
+### Additions
+
+- Added turn reserved IP for production turn servers of EU|US|IN.
```

Comment:
can we reframe this? as a user, i should i need to know if i have to make any changes. can we make it actionable?